<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpWidget</class>
 <widget class="QWidget" name="TcpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TcpWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QTextEdit" name="ReadMsg"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTextEdit" name="SendMsg"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnSend">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>312</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="btnDisconnect">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
